### ISSUE TITLE
Some issues with Dark Mode 

### ISSUE THREAD

**Issue by jackrua** [RIP-Comm/sossoldi#157]
If you set Dark Mode, the colors in the line chart do not change. 
This point is already addressed in the code: 

https://github.com/RIP-Comm/sossoldi/blob/fc3c316695686c0556543fac5da24b7b8b2b6676/lib/pages/home_page.dart#L139-L145

We could use something like `blue6` (`0xFF7DA1C4`) to get a nice contrast: 

![p3](https://github.com/RIP-Comm/sossoldi/assets/155536850/edab067f-9b94-4fc0-9f0e-61d9b3c3d26b)

Moreover all the account and category icons should be switched to Dark Mode in different pages (only in the settings page they are updated while, for example, in the home page they are not). 

Finally in the transactions page, if you switch to Dark Mode the month selector creates so much contrast that is almost unreadable: 

![p4](https://github.com/RIP-Comm/sossoldi/assets/155536850/c9ae3f40-855d-48e8-bcb2-a7a96adc6e25)

**Comment by theperu** [comment]
Hi! There is an open PR which addressed a few issues that were already found about the dark mode, if you want you could pick it up so that we can solve all the problems that we currently have for the theme

**Comment by diegodiome** [comment]
Can I work on this? There are still some issues, such as the transaction selector and line graph tooltip.

**Comment by theperu** [comment]
Yes @diegodiome, if you want you can also take a look at the open PR for the dark theme #124 which fixed some things but not all 😁
